addons/resource.language.en_gb/resources/strings.po

```diff
@@ -17115,7 +17115,18 @@ msgctxt "#36549"
 msgid "Use iOS8 compatible AirPlay support. If you have trouble with older iOS devices detecting this application as a valid target try switching this off. This option requires a restart to take effect!"
 msgstr ""
 
-#empty strings from id 36550 to 36599
+#: system/settings/settings.xml
+msgctxt "#36550"
+msgid "GUI stereo effect strength"
```

Comment:
well, it's a) in the GUI/Skin section and b) description explains. Also, adding GUI to the settings label might make it too long in translations. So the shorter, the better